### PR TITLE
Updating node image and go image in Dockerfile

### PR DESCRIPTION
**Pull request by Alevsk**
- Updating node image to 17
- Updating go image to 1.17

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -72,7 +72,7 @@ test-integration:
 	@(docker stop minio)
 
 test-permissions:
-	@(docker run -d --name minio --rm -p 9000:9000 quay.io/minio/minio:latest server /data{1...4})
+	@(docker run -d --name minio --rm -p 9000:9000 minio/minio:manual-test server /data{1...4})
```
Comment (harshavardhana) [inline Makefile:75]:
```suggestion
	@(docker run -d --name minio --rm -p 9000:9000 minio/minio:manual-test server /data{1...4})
```

is this intentional?

File: portal-ui/build/asset-manifest.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
     "static/js/1140.c920261b.chunk.js": "./static/js/1140.c920261b.chunk.js",
     "static/js/2094.d27076fb.chunk.js": "./static/js/2094.d27076fb.chunk.js",
     "static/js/7950.3a3eca23.chunk.js": "./static/js/7950.3a3eca23.chunk.js",
-    "static/js/671.f558f624.chunk.js": "./static/js/671.f558f624.chunk.js",
+    "static/js/671.484c5d87.chunk.js": "./static/js/671.484c5d87.chunk.js",
```
Comment (harshavardhana) [inline portal-ui/build/asset-manifest.json:51]: why build assets in this? @Alevsk

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: portal-ui/build/asset-manifest.json (hunk 2 above)
Comment (Alevsk) [reply portal-ui/build/asset-manifest.json]: fixed

**Review by harshavardhana — APPROVED** [review]

File: portal-ui/tests/constants/timestamp.txt [hunk 3]
```diff
@@ -1 +1 @@
-1642917498
+1642917498
```
Comment (harshavardhana) [inline portal-ui/tests/constants/timestamp.txt:1]: this looks like an incorrect file. @Alevsk

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: portal-ui/tests/constants/timestamp.txt (hunk 3 above)
Comment (dvaldivia) [reply portal-ui/tests/constants/timestamp.txt:1]: it gets regenerated every time you run tests locally

**Review by dvaldivia — APPROVED** [review]
LGTM.

File: portal-ui/tests/constants/timestamp.txt (hunk 3 above)
Comment (Alevsk) [reply portal-ui/tests/constants/timestamp.txt]: fixed

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**